bin/ci

```diff
@@ -67,6 +67,7 @@ prepare_build() {
 
   on_osx brew update
   on_osx brew install crystal-lang
+  with_build_env 'shards update'
```

Comment:
This should be `shards install` not `shards update`, because update will **not** install the dependencies based on the `shard.lock` file and only rely on the `shard.yml` file to get the version.
However it's important that the tests use the version specified in `shard.lock`